### ISSUE TITLE
Tests are error'ing

### ISSUE THREAD

**Issue by RafaelKeramidas** [xz64/license-webpack-plugin#7]
Hi,

I cloned the project and tried to run the tests but I hit an error: 
```
> license-webpack-plugin@0.3.0 test C:\Users\myusername\Documents\GitHub\license-webpack-plugin
> tape test/*.js | tap-diff

C:\Users\myusername\Documents\GitHub\license-webpack-plugin\test\index.test.js:303
    t.equal(plugin.modules[0].licenseText, 'text: MIT');
                             ^

TypeError: Cannot read property 'licenseText' of undefined
    at C:\Users\myusername\Documents\GitHub\license-webpack-plugin\test\index.test.js:303:30
    at C:\Users\myusername\Documents\GitHub\license-webpack-plugin\node_modules\tapes\index.js:82:7
    at C:\Users\myusername\Documents\GitHub\license-webpack-plugin\node_modules\async\lib\async.js:52:16
    at iterate (C:\Users\myusername\Documents\GitHub\license-webpack-plugin\node_modules\async\lib\async.js:260:24)
    at Object.async.forEachOfSeries.async.eachOfSeries (C:\Users\myusername\Documents\GitHub\license-webpack-plugin\node_modules\async\lib\async.js:281:9)
    at Object.async.forEachSeries.async.eachSeries (C:\Users\myusername\Documents\GitHub\license-webpack-plugin\node_modules\async\lib\async.js:214:22)
    at runWrapperFns (C:\Users\myusername\Documents\GitHub\license-webpack-plugin\node_modules\tapes\index.js:90:9)
    at Test.<anonymous> (C:\Users\myusername\Documents\GitHub\license-webpack-plugin\node_modules\tapes\index.js:81:5)
    at Test.bound [as _cb] (C:\Users\myusername\Documents\GitHub\license-webpack-plugin\node_modules\tape\lib\test.js:65:32)
    at Test.run (C:\Users\myusername\Documents\GitHub\license-webpack-plugin\node_modules\tape\lib\test.js:84:10)
```

I ran the following command: `npm install && npm run test`

I'm on Windows 7 x64 using NodeJS 6.9.1 and NPM 3.8.6.

Unfortunately I'm too busy to look into it myself, otherwise I would try to fix it and do a PR :-)

Cheers,
Raf

**Comment by xz64** [comment]
@RafaelKeramidas I've pushed a change to the dev branch for this. Try it out and let me know how it goes. Thanks.

**Comment by RafaelKeramidas** [comment]
Thanks for the fix @xz64 ! Tested & working (all tests passing) on the dev branch.

Cheers,
Raf

**Comment by xz64** [comment]
Fix published in v0.4.0
